infra/network.yaml

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: ''
+Description: 'The VPC for the UAT.  One region, to save costs, but two AZs for reliability.  Parallel stacks - two public and two private subnets.'
```

Comment:
suggestion (blocking): Expand AZ and VPC acronyms for readability (aka I do not know what they mean :) )

Reply:
Done!  Availability Zone and Virtual Private Cloud.